From b3d5429449aef61cb8926c33925fd7b5cad737de Mon Sep 17 00:00:00 2001
From: Becky Bruce <beckyb@kernel.crashing.org>
Date: Tue, 19 Apr 2011 19:06:16 -0500
Subject: [PATCH 2/2] tests/run_tests.py: fix typo in test invocation.

One test was being invoked with HUGETLB_MORECORE=es (instead of yes).....

Signed-off-by: Becky Bruce <beckyb@kernel.crashing.org>
---
 tests/run_tests.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tests/run_tests.py b/tests/run_tests.py
index 8055940..f0049b3 100755
--- a/tests/run_tests.py
+++ b/tests/run_tests.py
@@ -567,7 +567,7 @@ def functional_tests():
     do_test("heapshrink", LD_PRELOAD="libhugetlbfs.so libheapshrink.so",
             HUGETLB_MORECORE="yes")
     do_test("heapshrink", LD_PRELOAD="libheapshrink.so", HUGETLB_MORECORE="yes",
-            HUGETLB_MORECORE_SHRINK="es")
+            HUGETLB_MORECORE_SHRINK="yes")
     do_test("heapshrink", LD_PRELOAD="libhugetlbfs.so libheapshrink.so",
             HUGETLB_MORECORE="yes", HUGETLB_MORECORE_SHRINK="yes")
     do_test("heap-overflow", HUGETLB_VERBOSE="1", HUGETLB_MORECORE="yes")
-- 
1.5.6.5

